src/main/java/org/skriptlang/skript/bukkit/InputKey.java

```diff
@@ -0,0 +1,50 @@
+package org.skriptlang.skript.bukkit;
```

Comment:
might be better under `bukkit.util`

though, I don't think it would be a bad idea to put all of this in a module `bukkit.input` or `bukkit.player.input`